src/constructs/ec2/vpc.ts

```diff
@@ -40,7 +49,7 @@ export class GuVpc {
   static subnetsfromParameter(scope: GuStack, props?: GuSubnetProps): ISubnet[] {
     const type = props?.type ?? SubnetType.PRIVATE;
 
-    const subnets = new GuSubnetListParameter(scope, `${type}Subnets`, {
+    const subnets = new GuSubnetListParameter(scope, `${maybeApp(props)}${type}Subnets`, {
```

Comment:
[`AppIdentity.suffixText`](https://github.com/guardian/cdk/blob/d69bdbe181d90f0f2c502dee9b05095f1402301a/src/constructs/core/identity.ts#L16) in other places that are app aware, can this be reused here for consistency?

Reply:
As discussed IRL, this is not a trivial change and so we've elected to leave this for a future PR.